### PR TITLE
Adding parse method for json and change some code in Api

### PR DESCRIPTION
**Pull request by cihanozluk**

### CODE REVIEW

File: Api/routes/words.js [hunk 1]
```diff
@@ -1,65 +1,43 @@
...
+  res.send("Get posts");
+});
+
+router.get(
+  "/:word/:language/:partofspeech/:numberofcluster",
+  async (req, res) => {
+    try {
+      let options = {
+        mode: "text",
+        pythonOptions: ["-u"], // get print results in real-time
```
Comment (rebelDi) [inline Api/routes/words.js:17]: Why do we need to print result in real-time?

File: Api/routes/words.js [hunk 2]
```diff
@@ -1,65 +1,43 @@
...
+          req.params.partofspeech,
+          req.params.numberofcluster,
+        ],
+      };
+
+      PythonShell.run("main.py", options, function (err, results) {
+        if (err) throw err;
+        // results is an array consisting of messages collected during execution 
+        // results = JSON.stringify(results)
+        res.send(results.toString().replace(/\\|\//g,'').replace(/'/g,'"'));
```
Comment (rebelDi) [inline Api/routes/words.js:31]: We need to add replacement for \r \n characters, just to be safe

File: src/components/Definitions/Definitions.js [hunk 3]
```diff
@@ -2,37 +2,31 @@ import React from "react";
 import "./Definitions.css";
 
 const Definitions = ({ meanings, word, language, partOfSpeech }) => {
-  console.log(meanings, word, language, partOfSpeech);
+  console.log(word, language, partOfSpeech);
+  console.log('meaningssss',meanings);
```
Comment (rebelDi) [inline src/components/Definitions/Definitions.js:6]: Do we need these logs just for debugging?

**Review by rebelDi — APPROVED** [review]
Looks good!

File: Api/routes/words.js (hunk 2 above)
Comment (cihanozluk) [reply Api/routes/words.js:31]: makes sense I will do it

File: src/components/Definitions/Definitions.js (hunk 3 above)
Comment (cihanozluk) [reply src/components/Definitions/Definitions.js:6]: no I should have removed them

File: Api/routes/words.js (hunk 1 above)
Comment (cihanozluk) [reply Api/routes/words.js:17]: it is a shell script to read messages which comes from python file in real-time

**Pull request merged**